apps/settings/js/panels/simcard_manager/simcard_manager.js

```diff
@@ -276,26 +276,21 @@ define(function(require) {
     },
 
     /**
-     * Update SimSecurityUI
+     * Update the UI of the sim settings section
      *
      * @memberOf SimCardManager
      * @access private
      */
-    _updateSimSecurityUI: function scm__updateSimSecurityUI() {
+    _updateSimSettingsUI: function scm__updateSimSettingsUI() {
       var firstCardInfo = this._simcards[0].getInfo();
       var secondCardInfo = this._simcards[1].getInfo();
+      var hidden = firstCardInfo.absent && secondCardInfo.absent ||
+        this._isAirplaneMode;
 
       // if we don't have any card available right now
       // or if we are in airplane mode
-      if (firstCardInfo.absent && secondCardInfo.absent ||
-        this._isAirplaneMode) {
-          this._elements.securityEntry.setAttribute('aria-disabled', true);
-          this._elements.securityDesc.setAttribute('data-l10n-id', 'noSimCard');
-      } else {
-        this._elements.securityEntry.setAttribute('aria-disabled', false);
-        this._elements.securityDesc.removeAttribute('data-l10n-id');
-        this._elements.securityDesc.textContent = '';
-      }
```

Comment:
Let's see whether we should handle the APM case here base on Jenny's response later.